src/features/projects/components/ProjectSnippet.tsx

```diff
@@ -35,7 +36,11 @@ export default function ProjectSnippet({
   const router = useRouter();
   const { t, i18n, ready } = useTranslation(['donate', 'common', 'country']);
   const { embed, callbackUrl } = React.useContext(ParamsContext);
-  const popupState = usePopupState({
+  const popupState1 = usePopupState({
```

Comment:
Avoid using names like `popupState1` and `popupState2`. It would be clearer if you identified the popup with the name e.g. `verifiedPopupState`